source/help/policies/content-types.md

```diff
@@ -87,6 +87,10 @@ Requirements for Comments and Replies:
 - "Comments-on" content type must have a Title: "Comment(s) on '<title-of-original-paper>'" and the Comments field must contain "comment(s) on <arXiv-ID>".
 - A response from the original authors is always permitted and must have a Title "Reply to comment(s) on '<title-of-original-paper>'" and the Comments field must contain "reply to comment(s) on <arXiv-ID>" in the comment field.
```

Comment:
To match above, add a colon after "Title", "... must have a Title: "Reply to...". 

For clarification and to avoid repetition, "...the Comments field must contain "reply to comment(s) on <arXiv-ID>" in the comment field" could instead read, "the metadata Comments field must contain "reply to comment(s) on <arXiv-ID>".